lib/bolt_server/transport_app.rb

```diff
@@ -675,6 +681,56 @@ def make_winrm_target(target_hash)
       end
     end
 
+    # Returns the base64 encoded tar archive of plugin code that is needed to calculate
+    # custom facts
+    #
+    # @param versioned_project [String] the versioned_project to build the plugin tarball from
+    post '/project_facts_plugin_tarball' do
```

Comment:
In the ticket I specified this as `GET` why did we implement with POST?

Reply:
Oh yeah, in case we extend this with a request body that may include additional input (similar to why project_inventory_targets was turned into a POST endpoint since GET w/ request bodies are weird). My gut feeling tells me that could be a possibility, but it is also a gut feeling so happy to switch back to a GET.

Reply:
I vote GET, but based on ^^ that comment noticed that `project_inventory_targets` is a POST but retained the query param. We should move that into the POST body at some point.

Reply:
Ok, updated to GET.